com.unity.netcode.gameobjects/Components/NetworkAnimator.cs

```diff
@@ -209,6 +208,42 @@ public Animator Animator
             }
         }
 
+        [Tooltip("When enabled, the NetworkAnimator will operate in server authoritative mode.")]
+        [SerializeField]
+        private bool m_IsServerAuthoritative;
```

Comment:
Should this be set to true by default as to not create a breaking change?

Reply:
Good question.  If we want to make server authoritative the default that is easy enough to do.
Do we want users to default to the server authoritative model or client authoritative model?
_(both approaches have pro's and con's)_

Reply:
+1 for making server-auth the default to not change the current default.